lms/djangoapps/grades/config/__init__.py

```diff
@@ -6,7 +8,12 @@ def assume_zero_if_absent(course_key):
     """
     Returns whether an absent grade should be assumed to be zero.
     """
-    return should_persist_grades(course_key) and waffle_func().is_enabled(ASSUME_ZERO_GRADE_IF_ABSENT)
+    return (
+        should_persist_grades(course_key) and (
+            settings.FEATURES.get('ASSUME_ZERO_GRADE_IF_ABSENT_FOR_ALL_TESTS') or
+            waffle_func().is_enabled(ASSUME_ZERO_GRADE_IF_ABSENT)
```

Comment:
We'll have to keep this waffle switch around long-term for openedX instances that have persistent grades on but haven't backfilled yet, right? Or is this one eventually going away too?

Reply:
Yes, exactly.  We'll need to keep this one around for backfilling.  I had updated this ticket with what I believe needs to be kept and what needs to be removed: https://openedx.atlassian.net/browse/EDUCATOR-1396
It would be good if you review that as well to see if I missed anything.